scripts/common/deploy-system.ts

```diff
@@ -1,4 +1,12 @@
-import { AutomationBot, AutomationExecutor, CloseCommand, McdUtils, McdView, ServiceRegistry } from '../../typechain'
+import {
```

Comment:
It's good to have our deployment-system script synced, but we need an incremental deployment script as well (or maybe instead - think like DB migrations). I would suggest you create a script which 
1) deploys AutomationSwap
2) fetch list of all Executor legal callers
3) add those callers as legal caller of a Swap as well
4) adds AutomationSwap instance as a legal Executor caller